### DISCUSSION TITLE
sbt stuck (this may take some time)

### DISCUSSION THREAD

**Discussion by oxcar** [sbt/sbt#6958] [Q&A]
I just downloade sbt with brew (MacOS) and tried to start a project, and it seems to be stuck for some reason. This never happened to me in the past. I launched sbt with -v and in the console it reads:

`[info] [launcher] getting org.scala-sbt sbt 1.6.2  (this may take some time)...`

I tried several times, and waited for more than 20-30 minutes, but nothing happened

ps: I have an M1 processor

**Comment by SethTisue** [comment]
a JVM thread dump might contain some clues?

**Reply by oxcar** [reply]
I restarted my computer and it started working. ** head scratching **
